test/metabase/models/user_test.clj

```diff
@@ -65,7 +65,7 @@
         (try
           (if google-auth?
             (user/create-new-google-auth-user! new-user-first-name new-user-last-name new-user-email)
-            (user/invite-user!                 new-user-first-name new-user-last-name new-user-email password invitor))
+            (user/invite-user!                 new-user-first-name new-user-last-name new-user-email password nil invitor))
```

Comment:
yeah I think this confirms to me at least that keyword args are the way to go here since adding new ones doesn't require changing every single invocation